apps/pie-storybook/stories/pie-assistive-text.stories.ts

```diff
@@ -49,7 +49,7 @@ const Template : TemplateFunction<AssistiveTextProps> = ({
     slot,
 }) => html`
     <pie-assistive-text
-        variant="${variant}">
+        variant="${variant || defaultProps.variant}">
```

Comment:
Would we be able to use something like `ifDefined` here instead? I would expect the assistive text component to handle the default variant if it isn't provided
```suggestion
        variant="${ifDefined(variant)}">
```

Reply:
Also I noticed that in pie-card's story file, the HTML template is missing either `defaultProps` or `isDefined`

Reply:
Well noticed! 👍